src/System.Private.Xml/src/System/Xml/Serialization/Compiler.cs

```diff
@@ -27,9 +27,6 @@ namespace System.Xml.Serialization
 
     internal class Compiler
     {
-#if !XMLSERIALIZERGENERATOR
-        private bool _debugEnabled = DiagnosticsSwitches.KeepTempFiles.Enabled;
-#endif
```

Comment:
@sepidehMS Thanks, I think this means that the code is shared between projects and thus my tool is operating in an unsafe manner and shouldn't have detected this. i.e. I have some work to do